src/test/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImplTest.java

```diff
@@ -34,26 +43,41 @@ public class TextTranslationServiceImplTest extends AbstractTest {
     @Mock
     private AutocompleteService autocompleteService;
 
-    public TextTranslationServiceImplTest() {
+    @Mock
+    private GCSStore gcsStore;
+
+    @Mock
+    private StoreOntologyService storeOntologyService = getStorageServiceMock();
+
+    public TextTranslationServiceImplTest() throws IOException {
```

Comment:
If you remove line 50, you can revert this line so it doesn't throw an exception